docker_ci/Dockerfile

```diff
@@ -129,6 +132,7 @@ USER dev
 COPY --chown=dev:dev . ${MOOSE_DIR}
 
 ARG BUILD_MODULES
+ARG MOOSE_METHODS=opt
```

Comment:
Is there a reason to have moose and libmesh methods separately? This leaves room for error as moose methods always have to be a subset of libmesh methods :-/